report/renderer/util.js

```diff
@@ -212,9 +212,39 @@ class Util {
       });
     }
 
+    // In 10.0, full-page-screenshot became a top-level property on the LHR.
+    if (clone.audits['full-page-screenshot']) {
```

Comment:
Can you add some backcompat unit tests for these?

Reply:
Yeah, but I think it'd be best to add an assertion to one of the tests coming up in https://github.com/GoogleChrome/lighthouse/pull/14617

Reply:
We have several existing unit tests for backcompat though https://github.com/GoogleChrome/lighthouse/blob/main/report/test/renderer/util-test.js#L90